Songify/postman_collection.json

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
Missing test:
- POST with invalid JSON in payload e.g. something is missing or wrongly spelled

Reply:
added 4 new tests with 
- invalid JSON Payload
- missing value
- missing json member and value
- wrongly spelled json member

Example wrongly spelled: 

`curl --location --request POST 'https://kbe-songify.herokuapp.com/Songify/songs' \
--header 'Content-Type: application/json' \
--data-raw '{
    "titl": "Wrecking Ball",
    "artist": "MILEY CYRUS",
    "label": "RCA",
    "relased": 2013
}'`